src/Couchbase.Lite.Shared/API/Database/Database.cs

```diff
@@ -156,12 +156,21 @@ public sealed unsafe partial class Database : ICollection, IQueryFactory, IDispo
         private IJsonSerializer _jsonSerializer;
 #endif
 
+        private C4Database* _c4db;
         private C4CollectionObserver* _obs;
+
         private GCHandle _obsContext;
-        private C4Database* _c4db;
         private bool _isClosing;
         private ManualResetEventSlim _closeCondition = new ManualResetEventSlim(true);
 
+        //Pre 3.1 Database's Collection
+        private ICollection _defaultCollection = null;
+
+        //3.1+ Database
+        private ICollection _selectedCollection = null;
+        private Scope _selectedScope = null;
```

Comment:
There is no concept of "selected" scopes and collections in a database, why are these two here?